pkg/testutils/noncemanager/nonceManager.go

```diff
@@ -0,0 +1,90 @@
+package noncemanager
```

Comment:
@mkysel I moved this to a test util when I was thinking about using it for the loadtest. But I was getting a lot of errors which all went away when I switched to the database-backed nonce manager. Maybe a cause of some test flakes?